core/src/main/java/eu/fasten/core/data/JavaNode.java

```diff
@@ -62,4 +62,28 @@ public FastenURI changeName(final String className, final String methodName) {
         final var uri = this.getUri().toString().replace("/" + getClassName() + ".", "/" + className + ".");
         return FastenURI.create(uri.replace("." + getMethodName() + "(", "." + methodName + "("));
     }
+
```

Comment:
I have the same kind of comment as with `ExtendedRevisionJavaCallGraph`

Reply:
The same answer also applies here.